Rationale.md

```diff
@@ -162,9 +173,9 @@ variables by creating a separate stack data structure within linear memory. This
 stack is sometimes called the "aliased" stack, since it is used for variables
 which may be pointed to by pointers.
 
-This prevents WebAssembly from performing clever optimizations on the stack and
-liveness of such variables, but this loss isn't expected to be
-consequential. Common C compiler optimizations such as LLVM's global value
+This obstructs WebAssembly from performing clever optimizations on the stack and
```

Comment:
Can you reword this paragraph to be more direct? Something like:

Since the aliased stack appears to the WebAssembly engine as normal memory, WebAssembly optimizations that would target the aliased stack need to be more general, and thus more complicated. We observe that common compiler optimizations on the producer side, such as LLVM's global value numbering, effectively split address-taken variables into many small ranges that can often be allocated as local variables. Thus our expectation that any loss of optimization potential here is minimal.